src/llm_interface.jl

```diff
@@ -148,6 +148,17 @@ Requires two environment variables to be set:
 """
 struct DatabricksOpenAISchema <: AbstractOpenAISchema end
 
+"""
+   AzureOpenAISchema
+
+AzureOpenAISchema() allows user to call Azure OpenAI API. [API Reference](https://learn.microsoft.com/en-us/azure/ai-services/openai/reference)
+
+Requires two environment variables to be set:
+- `AZURE_OPENAI_API_KEY`: Azure token
+- `AZURE_OPENAI_ENDPOINT`: Address of the Azure resource (`"https://<resource>.openai.azure.com/"`)
```

Comment:
Could you please change the ENV variable name to `AZURE_OPENAI_HOST` (to match DATABRICKS_HOST)? It would make it easier for the user to find the right variables and `endpoint` is technically the full URL of the HOST + the endpoint you're calling.